Tzkt.Api/Websocket/Processors/OperationsProcessor.cs

```diff
@@ -184,6 +185,18 @@ void AddRange(HashSet<string> subs, IEnumerable<Operation> operations)
                     }
                 }
 
+                void AddTransaction(Sub sub, Models.TransactionOperation op)
```

Comment:
Probably better to pass `HashSet<string> subs` instead of `Sub sub` so we don't have to know its structure inside the function (in particular, that we must check exactly the `sub.Addresses` prop).